databuilder/extractor/dashboard/tableau/tableau_dashboard_last_modified_extractor.py

```diff
@@ -0,0 +1,128 @@
+import logging
+
+from pyhocon import ConfigTree, ConfigFactory  # noqa: F401
+from typing import Any  # noqa: F401
+
+from databuilder import Scoped
+
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.restapi.rest_api_extractor import STATIC_RECORD_DICT
+
+import databuilder.extractor.dashboard.tableau.tableau_dashboard_constants as const
+from databuilder.extractor.dashboard.tableau.tableau_dashboard_utils import TableauDashboardAuth,\
+    TableauGraphQLApiExtractor, TableauDashboardUtils
+
+from databuilder.rest_api.rest_api_query import RestApiQuery  # noqa: F401
+from databuilder.rest_api.base_rest_api_query import BaseRestApiQuery  # noqa: F401
```

Comment:
These imports are not used

Reply:
I was wondering how I kept missing these, I think that `noqa: F401` comment hides that unused error from the linter. I actually can't check that right now, cause `flake8` is not happy about something:
```
Traceback (most recent call last):
  File "/Users/carter.landis/workspace/virtualenvs/contrib-databuilder/bin/flake8", line 8, in <module>
    sys.exit(main())
  File "/Users/carter.landis/workspace/virtualenvs/contrib-databuilder/lib/python3.8/site-packages/flake8/main/cli.py", line 16, in main
    app.run(argv)
  File "/Users/carter.landis/workspace/virtualenvs/contrib-databuilder/lib/python3.8/site-packages/flake8/main/application.py", line 396, in run
    self._run(argv)
  File "/Users/carter.landis/workspace/virtualenvs/contrib-databuilder/lib/python3.8/site-packages/flake8/main/application.py", line 384, in _run
    self.run_checks()
  File "/Users/carter.landis/workspace/virtualenvs/contrib-databuilder/lib/python3.8/site-packages/flake8/main/application.py", line 310, in run_checks
    self.file_checker_manager.run()
  File "/Users/carter.landis/workspace/virtualenvs/contrib-databuilder/lib/python3.8/site-packages/flake8/checker.py", line 319, in run
    self.run_parallel()
  File "/Users/carter.landis/workspace/virtualenvs/contrib-databuilder/lib/python3.8/site-packages/flake8/checker.py", line 288, in run_parallel
    for ret in pool_map:
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/multiprocessing/pool.py", line 448, in <genexpr>
    return (item for chunk in result for item in chunk)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/multiprocessing/pool.py", line 868, in next
    raise value
AttributeError: 'FlakesChecker' object has no attribute 'CONSTANT'
```
Any idea what might be up with this error?

Reply:
^ happens no matter what I pass to `flake8`, whether it's `.` or a folder or a single file

Reply:
I can't reproduce that error. I do notice that your python version is 3.8. I'm on 3.7.5, so that _could_ be it

Reply:
Switching to 3.7 seemed to resolve this issue